### ISSUE TITLE
My youtube-dl downloads are being throttled/limited to about 80 kbs after some time.

### ISSUE THREAD

**Issue by Githubdude15** [ytdl-org/youtube-dl#29316]
## Checklist

- [x] I'm asking a question
- [x] I've looked through the README and FAQ for similar questions
- [x] I've searched the bugtracker for similar questions including closed ones

## Question

WRITE QUESTION HERE

I am using youtube-dl for a few years now. I never encountered this. After some months i started using it again. First download speed is normal and after a while there is a download limit. I am using the current version.

I also tried it with the aria2c, but no success.

`youtube-dl https://www.youtube.com/playlist?list=UUUllb83Lugbka-rvF6Nutxg -f bestvideo+bestaudio --external-downloader aria2c --external-downloader-args "-j 16 -s 16 -x 16 -k 5M"`

![Screenshot (5)](https://user-images.githubusercontent.com/85956711/122205764-e363f480-cea0-11eb-9e60-9dd13bce5099.png)
![Screenshot (1)](https://user-images.githubusercontent.com/85956711/122205768-e3fc8b00-cea0-11eb-9f39-7564402480be.png)

Also, when youtube-dl was taken down due to RIAA. I had some youtube.exe files and tried using them, with an updated and an older version. Both were completely blocked by youtube. It did not work at all. How is this even possible. Can google reduce the download speed of youtube-dl or block it completely?

**Comment by dstftw** [comment]
Read new issue template and fill the correct template.

**Comment by Githubdude15** [comment]
I think it fixed itself. I'm running it now for about a day with fullspeed. I don't know what it was.
